requirements/edx/github.txt

```diff
@@ -71,7 +71,7 @@ git+https://github.com/edx/pa11ycrawler.git@0.0.4#egg=pa11ycrawler==0.0.4
 
 # Our libraries:
 git+https://github.com/edx/XBlock.git@xblock-0.4.10#egg=XBlock==0.4.10
--e git+https://github.com/edx/codejail.git@6b17c33a89bef0ac510926b1d7fea2748b73aadd#egg=codejail
+-e git+https://github.com/edx/codejail.git@cdyer/multijail#egg=codejail==0.0.1
```

Comment:
pin or use `-e`, don't do both(lean towards pinning).  It won't necessarily break anything the way you have it(until the version changes in the setup.py for you codejail branch) but we want to move away from using `-e` as we update these requirements.